.github/workflows/build.yaml

```diff
@@ -15,6 +15,7 @@ jobs:
         env:
           - ALPINE_VERSION=3.14 ROS_DISTRO=noetic
           - ALPINE_VERSION=3.17 ROS_DISTRO=noetic
+          - ALPINE_VERSION=3.17 ROS_DISTRO=humble
```

Comment:
I think it's better to explicitly have ROS_DISTRIBUTION_TYPE

Reply:
Done in 5cae833af9b03f5f557772cbbd80009490f78c9a